apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -530,6 +552,8 @@ type SystemSphinxSpec struct {
 	PriorityClassName *string `json:"priorityClassName,omitempty"`
 	// +optional
 	TopologySpreadConstraints []v1.TopologySpreadConstraint `json:"topologySpreadConstraints,omitempty"`
+	// +optional
+	Annotations map[string]string `json:"annotations,omitempty"`
```

Comment:
SystemSphinxSpec should be deprecated and we shouldn't be adding stuff here.

Reply:
fixed. Thank you